.github/workflows/swig-interface-gen.yml

```diff
@@ -43,3 +43,4 @@ jobs:
         GIT_EMAIL: "HELICS-bot@users.noreply.github.com"
         GIT_NAME: "HELICS-bot"
         BRANCH_PREFIX: "swig-gen/"
+        DISABLE_STAGING: true
```

Comment:
Do we do the stages in the script that generates the files?

Reply:
I added a bit to the action so that it stages changed files by default (should be the more common use case). The swig generating action disables staging because the step before stages just changed files in the `interfaces/` subfolder (maybe more conservative than needed).